lib/licensee/content_helper.rb

```diff
@@ -12,6 +12,18 @@ module ContentHelper
     MARKDOWN_HEADING_REGEX = /\A\s*#+/
     VERSION_REGEX = /\Aversion.*$/i
     MARKUP_REGEX = /[#_*=~\[\]()`|>]+/
+    CC_LEGAL_CODE_REGEX = /^\s*Creative Commons Legal Code\s*$/i
+    CC0_INFO = 'For more information, please see\s*' +
+               '<http://creativecommons.org/publicdomain/zero/1.0/>\s*'.freeze
+    CC0_INFO_REGEX = /#{CC0_INFO}/im
+    CC0_DISCLAIMER = %(CREATIVE COMMONS CORPORATION IS NOT A LAW FIRM AND DOES
```

Comment:
Can we dynamically pull this from the license text, e.g., via regex rather than hard coding it twice in the project?

Reply:
We could. Seems more complicated than just repeating, but could be used to handle other optional sections as well. Probably a good idea. I'll try adding in a bit.